python/cuml/linear_model/logistic_regression.pyx

```diff
@@ -476,17 +481,17 @@ class LogisticRegression(Base, ClassifierMixin):
         super(LogisticRegression, self).__init__(handle=None,
                                                  verbose=state["verbose"])
 
-        if "qn" in state:
-            qn = state["qn"]
-            if qn.coef_ is not None:
-                if qn.fit_intercept:
-                    state["coef_"] = qn.coef_[0:-1]
-                    state["intercept_"] = qn.coef_[-1]
+        if "solver_model" in state:
+            solver_model = state["solver_model"]
+            if solver_model.coef_ is not None:
+                if solver_model.fit_intercept:
+                    state["coef_"] = solver_model.coef_[0:-1]
+                    state["intercept_"] = solver_model.coef_[-1]
                 else:
-                    state["coef_"] = qn.coef_
-                    n_classes = qn.coef_.shape[1]
+                    state["coef_"] = solver_model.coef_
+                    n_classes = solver_model.coef_.shape[1]
                     state["intercept_"] = CumlArray.zeros(
-                        n_classes, dtype=qn.coef_.dtype
+                        n_classes, dtype=solver_model.coef_.dtype
```

Comment:
Maybe we can remove the set/getstate now with the base class solver logic in place?